UI/views/status.html.erb

```diff
@@ -24,6 +24,9 @@ Status URL Links:
   </table>
```

Comment:
Not related to current commit - but the tables in status.html.erb should really be ordered lists.